apps/client/src/main.tsx

```diff
@@ -0,0 +1,14 @@
+import { StrictMode } from 'react';
+import * as ReactDOM from 'react-dom/client';
+
+import App from './app/app';
+
+const root = ReactDOM.createRoot(
+	document.getElementById('root') as HTMLElement,
```

Comment:
use tabs for spacing.

Reply:
done

related commit - https://github.com/the-ai-team/razor/commit/1c04b19bd37ac93a17e362f3e71f53c4bf7c2bf3